### PR TITLE
[Imporvement][Doc] /development/architecture-design.html

### PR DESCRIPTION
**Pull request by Tandoy**
·Modify the zk node error in the framework design module distributed_lock.png of the developer guide
·There are some Chinese words in the picture of the modified English document
·Adjust the size of related pictures to make them look clearer

### CODE REVIEW

File: development/en-us/architecture-design.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Before explaining the architecture of the schedule system, let us first understa
 **DAG：** Full name Directed Acyclic Graph，referred to as DAG。Tasks in the workflow are assembled in the form of directed acyclic graphs, which are topologically traversed from nodes with zero indegrees of ingress until there are no successor nodes. For example, the following picture:
 
 <p align="center">
-  <img src="/img/dag_examples_cn.jpg" alt="dag示例"  width="60%" />
+  <img src="/img/architecture-design/dag_examples_en_us.PNG" alt="dag示例"  width="60%" />
```
Comment (ruanwenjun) [inline development/en-us/architecture-design.md:9]: Please use lowercase suffixes `dag_examples_en_us.png`

File: development/zh-cn/architecture-design.md [hunk 2]
```diff
@@ -131,12 +131,12 @@
 DolphinScheduler使用ZooKeeper分布式锁来实现同一时刻只有一台Master执行Scheduler，或者只有一台Worker执行任务的提交。
 1. 获取分布式锁的核心流程算法如下
  <p align="center">
-   <img src="https://analysys.github.io/easyscheduler_docs_cn/images/distributed_lock.png" alt="获取分布式锁流程"  width="50%" />
+   <img src="/img/architecture-design/distributed_lock_zh_cn.png" alt="获取分布式锁流程"  width="55%" />
```
Comment (ruanwenjun) [inline development/zh-cn/architecture-design.md:134]: Please rename to `distributed_lock_ch_zn.png`

**Review by ruanwenjun — CHANGES_REQUESTED** [review]

File: development/zh-cn/architecture-design.md (hunk 2 above)
Comment (Tandoy) [reply development/zh-cn/architecture-design.md:134]: ok,i will modify it.

**Comment by ruanwenjun** [comment]
@Tandoy Hi, there may be some changes needed.
image

image

And can we resize the image, it's too small and cannot read the words on it.

**Review by ruanwenjun — APPROVED** [review]
LGTM

File: development/zh-cn/architecture-design.md (hunk 2 above)
Comment (ruanwenjun) [reply development/zh-cn/architecture-design.md]: @Tandoy Sorry, please rename the file name to zh_cn, it's my mistake 😭

Comment (Tandoy) [reply development/zh-cn/architecture-design.md]: ok,i will fix them~

**Pull request merged**